docs/other/troubleshooting-gratia.md

```diff
@@ -0,0 +1,375 @@
+    gridftp-transfer/ProbeConfig:    EnableProbe="1"
+
+If you see no log files in `/var/log/gratia` you may have an error in the probe configuration file. Run manually the test for your probe (check `/etc/cron.d/gratia-probe-condor.cron`), e.g. `/usr/share/gratia/common/cron_check  /etc/gratia/condor/ProbeConfig`. If there is an error you may get a suggestion on where it is, e.g.:
+
+    :::console
+    root@host # /usr/share/gratia/common/cron_check  /etc/gratia/condor/ProbeConfig
+    Parse error in /etc/gratia/condor/ProbeConfig: not well-formed (invalid token): line 21, column 4
+
+Correct the error and restart gratia.
+
+Have you configured the resource names correctly? 
+-------------------------------------------------
+
+Do the names of your resources match the names in OIM?  Gratia retrieves the resource name from the `Site Information` section of the `/etc/osg/config.d/40-siteinfo.ini`
+
+
+``` file
+;===================================================================
+;                       Site Information
+;===================================================================
+
+[Site Information]
+; The group option indicates the group that the OSG site should be listed in,
+; for production sites this should be OSG, for vtb or itb testing it should be
+; OSG-ITB
+; 
+; YOU WILL NEED TO CHANGE THIS
+group = OSG
+
+; The host_name setting should give the host name of the CE  that is being 
+; configured, this setting must be a valid dns name that resolves
+; 
+; YOU WILL NEED TO CHANGE THIS
+host_name = tusker-gw1.unl.edu
+
+; The resource setting should be set to the same value as used in the OIM 
+; registration at the goc 
+; 
+; YOU WILL NEED TO CHANGE THIS
+resource = Tusker-CE1
+
+; The resource_group setting should be set to the same value as used in the OIM 
+; registration at the goc 
+; 
+; YOU WILL NEED TO CHANGE THIS
+resource_group = Tusker
+
+```
+
+Do those names match the names that you registered with OIM? If not, edit the names, and rerun "osg-configure -c".
+
+Did the site name change?
+-------------------------
+
+Was the site previously reporting data, but the site name (not host name, but site name) changed? When the site name changes, you need to ask the Gratia operations team to update the name of your site at the GRACC collector. To do this:
+
+1.  Open a ticket at [the GOC ticket web page](https://ticket.grid.iu.edu/goc/submit)
+2.  Select "Software or Service"
+3.  Select "GRACC Operations"
+4.  Type a friendly email that asks the GRACC team to change your site name at the collector. Make sure to tell them the old name and the new name.
+
+ Is a site reporting data?
+--------------------------
+
+You can see if the OSG GRACC Server is getting data from a site by going to [GRACC](https://gracc.opensciencegrid.org/dashboard/db/pilot-jobs-summary?orgId=1):
+
+1.  Specify the site name in Facility
+
+Condor must be configured to put information about each job into a special directory.
```

Comment:
Holy section title, batman. Maybe "Verifying Condor Job History Location"?

Also, should there be a note that this is for condor batch systems only?